libraries/Microsoft.Bot.Builder.AI.QnA/Dialogs/QnAMakerDialog.cs

```diff
@@ -290,6 +290,16 @@ public QnAMakerDialog([CallerFilePath] string sourceFilePath = "", [CallerLineNu
         [JsonProperty("rankerType")]
         public StringExpression RankerType { get; set; } = new StringExpression(RankerTypes.DefaultRankerType);
 
+        /// <summary>
+        /// Gets or sets the Maximum Score For LowScoreVariation to use.
+        /// </summary>
+        /// <value>
+        /// Max score for LowScoreVariation.
+        /// </value>
+        /// <seealso cref="RankerTypes"/>
+        [JsonProperty("maximumScoreForLowScoreVariation")]
+        public float MaximumScoreForLowScoreVariation { get => maximumScoreForLowScoreVariation; set => maximumScoreForLowScoreVariation = value; }
```

Comment:
>MaximumScoreForLowScoreVariation [](start = 21, length = 32)

You need to update MicrosoftQnAMAkerDialog.schema to add maximumScoreForLowScoreVariation